sonar-project.properties

```diff
@@ -0,0 +1,6 @@
+sonar.projectName=digital-marketplace
+sonar.sources=.
+sonar.sourceEncoding=UTF-8
+
+sonar.organization=<replace with your SonarCloud organization key>
+sonar.projectKey=<replace with the key generated when setting up the project on SonarCloud>
```

Comment:
change to `=digital_marketplace` - though this is a 'best guess' that they will name the sonar project after the name of the GH repo.